homeassistant/components/cover/knx.py

```diff
@@ -34,6 +39,9 @@
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_SETPOSITION_ADDRESS): cv.string,
     vol.Optional(CONF_INVERT_POSITION, default=False): cv.boolean,
+    vol.Optional(CONF_GETANGLE_ADDRESS): cv.string,
+    vol.Optional(CONF_SETANGLE_ADDRESS): cv.string,
```

Comment:
Thee options are optional. There are shutter with and without the angle feature. The code supports both now.